### PR TITLE
Audio text and haptics

### PR DESCRIPTION
**Pull request by Clarisa1999**
This PR implements audio text and haptics in the options page

### CODE REVIEW

**Comment by jaydeepsingh25** [comment]
@Clarisa1999 , can you please resolve the conflicts in this PR?

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "auditory-haptic-graphics-browser",
+  "name": "IMAGE-browser",
```
Comment (jaydeepsingh25) [inline package-lock.json:2]: it shouldn't be changed here, instead please add a name field in package.json with name as "IMAGE-browser-extension". Package.json will automaticlly reflect that.

File: src/_locales/en/messages.json [hunk 2]
```diff
@@ -24,7 +24,7 @@
         "description": "Header of popup window"
     },
     "renderingLabel": {
-        "message": "Interpretation",
+        "message": "Rendering",
```
Comment (jaydeepsingh25) [inline src/_locales/en/messages.json:27]: Any specific reason to change it to "Rendering"? it was specifically changed in PR: https://github.com/Shared-Reality-Lab/IMAGE-browser/pull/135

**Review by jaydeepsingh25 — COMMENTED** [review]
@Clarisa1999 , Why only Audio Renderings are set to default? Is this the expected behaviour?
I think by default both Audio & text renderings should be selected.
Please make the necessary changes.

File: package-lock.json (hunk 1 above)
Comment (Clarisa1999) [reply package-lock.json:2]: @Cybernide from what I understand from the wireframe and our meetings, audio is the only one that is default right?

File: src/_locales/en/messages.json (hunk 2 above)
Comment (Clarisa1999) [reply src/_locales/en/messages.json:27]: > @Clarisa1999 , Why only Audio Renderings are set to default? Is this the expected behaviour? I think by default both Audio & text renderings should be selected. Please make the necessary changes.

@Cybernide from what I understand from the wireframe and our meetings, audio is the only one that is default right?

Comment (Clarisa1999) [reply src/_locales/en/messages.json:27]: Screen Shot 2022-02-09 at 4 33 01 PM

Comment (Clarisa1999) [reply src/_locales/en/messages.json:27]: I think text should be off and audio should be the default

**Review by jaydeepsingh25 — APPROVED** [review]

**Pull request merged**